slave-dotnet10/Dockerfile.rhel7

```diff
@@ -0,0 +1,44 @@
+FROM openshift/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet10-rhel7-docker" \
+      name="openshift3/jenkins-slave-dotnet10-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="1" \
+      io.k8s.display-name="Jenkins Slave .NET Core 1.0" \
+      io.k8s.description="The jenkins slave dotnet image has the dotnet tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,slave,dotnet,dotnet10"
+
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+# Install .NET Core
+RUN yum-config-manager --enable rhel-7-server-dotnet-rpms && \
+    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
+    yum-config-manager --disable epel >/dev/null || : && \
+    INSTALL_PKGS="rh-dotnetcore10 rh-nodejs4-npm" && \
```

Comment:
no (unless it's in centos itself).  we don't rely on it anymore, we do this instead:
https://github.com/openshift/jenkins/blob/master/slave-base/contrib/bin/run-jnlp-client#L18
https://github.com/openshift/jenkins/blob/master/slave-base/contrib/bin/generate_container_user#L7

which just adds the current container user to /etc/passwd during startup.

since you didn't touch the entrypoint here, that should "just work" for your image too.